### DISCUSSION TITLE
The "Trending Tab" is the reason I open github every day, please don't remove

### DISCUSSION THREAD

**Discussion by tinybug** [community/community#31682] [Other Feature Feedback, Questions, & Ideas]
Please keep the Trending tab

**Comment by shaohan0228** [comment]
if the cause is low usage: maybe github need to remove more features。

**Comment by elementh** [comment]
Same for me. Please reconsider.

**Comment by luzizhuo** [comment]
We are working on an open source tool called OSS Insight. https://github.com/pingcap/ossinsight 
Maybe you can consider using it to explore trending repos. https://ossinsight.io/#trending-repos

**Reply by mjablecnik** [reply]
It is nice alternative but unfortunately there I cannot choose language which I am using (Dart).
So it is not useful for me.. :-/

**Comment by pmg1991** [comment]
Please keep "Trending Repositories and Trending Developers

**Comment by jhribal** [comment]
Well i am using trending tab every day. Not sure how to trace open source projects on strike now.

Edit: I remember telling about it to one of my coworkers and he didn't know this feature exists so maybe promote it a bit.

**Comment by carlsverre** [comment]
I also use this tab every day and would be extremely sad to not have access to it. Is it being replaced by something better?
